snapshot.yaml

```diff
@@ -102,10 +102,6 @@
   sha: 829cac07c8e155cc1844dc3a069dcf9283abf603
   tag: None
   url: https://github.com/sherlock-project/sherlock
-- id: n2x1PPtw
```

Comment:
Please note that it's unnecessary to remove meta of unassimilable repos after #14